docker/package/package_generator.py

```diff
@@ -7,6 +7,61 @@
 from .packages import packages
 from .ubuntu import build_ubuntu_package
 
+def get_ubuntu_run_deps(args):
+    """
+    List all the ubuntu run dependencies. Return an empty list when using prebuilt static binaries.
+    """
+    if args.binaries_dir:
+        return []
+
+    return [
+        "libev-dev",
+        "libgmp-dev",
+        "libhidapi-dev",
+        "libffi-dev",
+        "zlib1g-dev",
+        "libpq-dev",
+    ]
+
+def get_fedora_run_deps(args):
+    """
+    List all the fedora run dependencies. Return an empty list when using prebuilt static binaries.
+    """
+    if args.binaries_dir:
+        return []
+
+    return [
+        "libev-devel",
+        "gmp-devel",
+        "hidapi-devel",
+        "libffi-devel",
+        "zlib-devel",
+        "libpq-devel",
+    ]
+
+
+def get_build_deps(args):
+    """
+    List all the common build dependencies. Return an empty list when using prebuilt static binaries.
+    """
+    if args.binaries_dir:
+        return []
+
+    return [
+        "make",
```

Comment:
I believe that `make` is still needed for the case when static binaries are used

Reply:
Yes, I've also included `wget` for the sapling package.